Vagrant/bootstrap.sh

```diff
@@ -215,7 +276,29 @@ if [ "$app_env" == "production" ]; then
     $RUN_AS_INSTALLUSER RAILS_ENV=production bundle exec rake assets:precompile
     $RUN_AS_INSTALLUSER RAILS_ENV=production bundle exec rake datarepo:setup_defaults
 fi
+# Fix up configuration files
+# 1. FITS
+$RUN_AS_INSTALLUSER sed -i "s@config.fits_path = \".*\"@config.fits_path = \"$fitsdir/$fitsver/fits.sh\"@" config/initializers/sufia.rb
+# 2. Create Sufia Solr core
+cd $SOLR_DATA
+$RUN_AS_SOLR_USER mkdir -p $app_env/conf # We name the cores after their environment
+$RUN_AS_SOLR_USER echo "name=$app_env" > $app_env/core.properties
+install -o $SOLR_USER -m 444 $hydradir/solr_conf/conf/solrconfig.xml $app_env/conf/solrconfig.xml
+install -o $SOLR_USER -m 444 $hydradir/solr_conf/conf/schema.xml $app_env/conf/schema.xml
+# Make links to keep the Hydra Solr solrconfig.xml paths happy
+$RUN_AS_SOLR_USER ln -s $SOLR_INSTALL/solr/contrib
+$RUN_AS_SOLR_USER ln -s $SOLR_INSTALL/solr/dist
+$RUN_AS_SOLR_USER mkdir lib
+$RUN_AS_SOLR_USER ln -s $SOLR_INSTALL/solr/contrib lib/contrib
+cd $hydradir
+# 3. Make the solr.yml file point to an appropriate $app_env core
+sed -i '/production:/ {N; s@^production:.*development@production:\n  url: http://localhost:8983/solr/production@}' config/solr.yml
+# 4. Make the blacklight.yml file point to an appropriate $app_env core
+sed -i '/production:/ {N; N; s@^production:\(.*\)/development@production:\1/production@}' config/blacklight.yml
+# 5. Make the fedora.yml point to Tomcat 7 port, not to hydra-jetty port 8983
+sed -i 's/url:\(.*\):8983/url:\1:8080/' config/fedora.yml
 $RUN_AS_INSTALLUSER bash "$hydradir/scripts/restart_resque.sh" "$app_env"
-$RUN_AS_INSTALLUSER bundle exec rake jetty:start
-# Start Nginx
+# Start services
+service tomcat7 start
+service solr start
```

Comment:
I left that out because, as we've discussed offline, arguably the Nginx shouldn't be restarted in the Vagrantfile runserver script.  All three services start at system boot, so restarting them there is redundant.